docker/README.md

```diff
@@ -0,0 +1,29 @@
+Build the image:
+```
+docker build -t mirror .
+```
+
+Start a *mirror server* on the current machine:
+```
+sudo docker run --rm --init -it -eU=$(id -u) -eG=$(id -g) -v $(pwd):/data --net=host mirror ./mirror server
```

Comment:
So what does the `$(pwd):/data` to? ...my guess is that it exposes the current working directory as `/data` within the container?

Reply:
Exactly! I will explain it with a bit more detail in the section to be added to the readme.